sci-biology/aghermann/ChangeLog

```diff
@@ -2,11 +2,5 @@
 # Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
 # $Header: $
 
-  18 Feb 2015; Justin Lecher <jlec@gentoo.org> aghermann-1.0.3.ebuild:
-  sci-biology/aghermann: Fix slot operators
-
-  18 Feb 2015; Justin Lecher <jlec@gentoo.org> aghermann-1.0.3.ebuild:
-  sci-biology/aghermann: Fix indention and use virtual for pkgconfig
-
-  17 Jan 2015; Andrei Zavada <johnhommer@gmail.com> aghermann-1.0.3.ebuild:
-  Initial packaging for gentoo.
+  19 Feb 2015; Andrei Zavada <johnhommer@gmail.com> aghermann-1.0.3.ebuild:
```

Comment:
Sorry, that was really accidental. During a rebase, I knew there should be a conflict on this file, and blindly went with the local version. It just didn't occur to me that someone did my work while I was busy sorting out the other PR.